docroot/WEB-INF/src/edu/osu/cws/evals/tests/BackendMgrTests.java

```diff
@@ -326,6 +326,43 @@ public void shouldSendProfFacultyEmailsRightAwayOnceReviewPeriodStarts() throws
         assert !BackendMgr.timeToSendFirstStatusEmail(appraisal, new DateTime());
     }
 
+    public void shouldUpdateSalaryWhenAppraisalIsDue() throws Exception {
+        Appraisal appraisal = new Appraisal();
+        appraisal.setJob(new Job());
+        appraisal.getJob().setAppointmentType(AppointmentType.CLASSIFIED_IT);
+        DateTime endDate = new DateTime().plusDays(61).withTimeAtStartOfDay();
+        appraisal.setEndDate(endDate.toDate());
+        appraisal.getSalaries().add(new Salary());
+
+        // get instance and call updateAppraisal
+        Session session = HibernateUtil.getCurrentSession();
+        Transaction tx = session.beginTransaction();
+        BackendMgr mgr = getMgrInstance();
+        session = HibernateUtil.getCurrentSession();
+        tx = session.beginTransaction();
```

Comment:
duplicate calls to `HibernateUtil.getCurrentSession()` and `session.beginTransaction()`